### PR TITLE
fix: match exchange rate decimal points

### PR DESCRIPTION
**Pull request by lcsfelix**

### CODE REVIEW

File: ui/src/app.js [hunk 1]
```diff
@@ -1243,7 +1244,7 @@ function formatFiat (amount, fractionDigits) {
     maximumFractionDigits: fractionDigits,
     minimumFractionDigits: fractionDigits
   })
-  return splitNumber(localized, jsLocaleCode) + ' ' + fiatCode
+  return splitNumber(localized, jsLocaleCode)
```
Comment (RafaelTaranto) [inline ui/src/app.js]: A lot of other functions rely on formatFiat for the fiat code concat, it should not be removed.

File: ui/src/app.js [hunk 2]
```diff
@@ -1257,11 +1258,11 @@ function setExchangeRate (_rates) {
   var cryptoToFiat = new BigNumber(rates.cashIn)
 
   var rateStr = formatFiat(cryptoToFiat.round(2).toNumber(), 2)
-  $('.crypto-rate-cash-in').html(`1 ${cryptoCode} = ${rateStr}`)
+  $('.crypto-rate-cash-in').html(`1 ${cryptoCode} = ${rateStr} ${fiatCode}`)
```
Comment (RafaelTaranto) [inline ui/src/app.js]: no need for this change here after undoing formatFiat

File: ui/src/app.js [hunk 3]
```diff
@@ -1257,11 +1258,11 @@ function setExchangeRate (_rates) {
...
 
   if (rates.cashOut) {
     var cashOut = new BigNumber(rates.cashOut)
-    var cashOutCryptoToFiat = cashOut && formatCrypto(cashOut.round(3).toNumber())
+    var cashOutCryptoToFiat = cashOut && formatFiat(cashOut.round(2).toNumber(), 2)
 
     $('.crypto-rate-cash-out').html(`1 ${cryptoCode} = ${cashOutCryptoToFiat} ${fiatCode}`)
```
Comment (RafaelTaranto) [inline ui/src/app.js]: No need for fiatCode here after undoing formatFiat

File: ui/src/app.js [hunk 4]
```diff
@@ -7,6 +7,7 @@ const SCREEN = params.get('screen')
 const DEBUG_MODE = SCREEN ? 'demo' : params.get('debug')
 const CASH_OUT_QR_COLOR = '#403c51'
 const CASH_IN_QR_COLOR = '#0e4160'
+const EXCHANGE_DP = 2
```
Comment (joshmh) [inline ui/src/app.js]: Is this used anywhere?

Comment (lcsfelix) [reply ui/src/app.js]: No, sorry, it was a leftover from my first solution.

**Pull request merged**